secant/Util/WalletStorageInteractor.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  WalletStorageInteractor.swift
+//  secant-testnet
+//
+//  Created by Lukáš Korba on 03/25/2022.
+//
+
+import Foundation
+import MnemonicSwift
+
+/// Representation of the wallet stored in the persistent storage (typically keychain, handled by `WalletStorage`).
+struct StoredWallet: Codable, Equatable {
+    var birthday: BlockHeight?
+    let language: MnemonicLanguageType
+    let seedPhrase: String
+    var hasUserPassedPhraseBackupTest: Bool
+    let version: Int
+}
```

Comment:
```suggestion
struct StoredWallet: Codable, Equatable {
    let language: MnemonicLanguageType
    let seedPhrase: String
    let version: Int
    
    var birthday: BlockHeight?
    var hasUserPassedPhraseBackupTest: Bool
}
```
Constants come before mutable properties according to our style guide 👍